internal/services/loganalytics/log_analytics_workspace_table_data_source.go

```diff
@@ -0,0 +1,103 @@
+package loganalytics
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/operationalinsights/2020-08-01/workspaces"
```

Comment:
Getting a test failure due to two different api versions being used. Lets use `2022-10-01` to keep it in line with the resource